submissions/adekemiAwomolo/PhpAndDtabase/customersView.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once('connection.php');
+    try{
+    global $pdo;
+    $sql = "SELECT single_orders.unit_price,
+    single_orders.quantity, 
+    single_orders.total_amount,
+    orders.id,
+    orders.created_at,
+    products_details.product_name,
+    customers_information.street_address,
+    customers_information.city,
+    customers_information.state,
+    customers_information.country
+    FROM single_orders
+    LEFT JOIN orders ON orders.id = single_orders.order_id
+    LEFT JOIN products_details ON products_details.id = single_orders.product_id
+    LEFT JOIN customers_information ON orders.customers_id = customers_information.customers_id";
+    $result = $pdo->query($sql);
+setcookie("customers_orders",time() +3600, "/");
+if(isset($_COOKIE["customers_orders"])){
```

Comment:
Also, kindly format your code block from line 21 below. I bare know where one `if` statement ends and where the other begins